LokiLabs/microsoft-graph-explorer-teams-app
# AB#38676 Cleaned up repo and removed unused and unnecessary files and folders

### Overview

This repository was full of unnecessary/useless files and required a clean up.


### Testing Instructions
* Just run the app from the root of the repository 
* Make sure everything works as it's should
